backend/tasks/robot_token.py

```diff
@@ -0,0 +1,67 @@
+import json
+from typing import Union
+
+from flask import make_response
+from flask_login import login_required
+from flask_restful import Resource, reqparse
+
+from utils import log_response, verify_admin_permissions
+
+from data.model import user, organization, InvalidOrganizationException, DataModelException, InvalidRobotException
+
+
+class RobotTokenTask(Resource):
+    @log_response
+    @verify_admin_permissions
+    @login_required
+    def post(self):
+        parser = reqparse.RequestParser()
+        parser.add_argument("token", type=str, help="raw token")
+        parser.add_argument("name", type=str, help="token name")
+        parser.add_argument("organization", type=str, help="organization name")
+        args = parser.parse_args()
+
+        robot_name = args.get('name')
```

Comment:
It needs to be `args.get('robot_name')` here. Frontend sends the key `robot_name`.